ui/oopsyraidsy/data/03-hw/raid/a12n.js

```diff
@@ -44,8 +44,8 @@ export default {
     {
       id: 'A12N Assault Cleanup',
       netRegex: NetRegexes.gainsEffect({ effectId: '461' }),
-      suppressSeconds: 5,
       delaySeconds: 20,
+      suppressSeconds: 5,
```

Comment:
This is a bit concerning, but I'm not sure of a great way (other than the obvious splitting ESLint rules across oopsyraidsy and raidboss) to prevent this on oopsy triggers that have all fields that are otherwise valid for raidboss. Also, it only happened twice.

Reply:
Why is this concerning?

Also, I could see some argument in splitting the rules (or having an option for the field ordering that gets moved to the eslint config?), since oopsy and raidboss do have different fields.  Otherwise, is the plan to put `mistake` and `deathReason` in that list?

Reply:
I mean, theoretically the list could contain the superset of both, but that (I feel) would become pretty difficult to parse. I can add an optional override so they can be done per raidboss/oopsy, but then that would be cloning the overrides block in eslintrc (not really an issue, but it’s a thing)

I don’t know if there’s an easy way to tell if we’re in an oopsy or raidboss trigger (other than fields or filepath) that we could key off of

Reply:
Yeah, that's why I was suggesting doing it in the eslint config.  You could pass the `orderList` in as an eslint option to the rule, and then apply a rule + different option to each directory.

I think we could have a "applies to both" and then a "applies to raidboss" and a "applies to oopsy" section so there's not a duplication of the entire overrides block.